loom/script/runtime/lsLuaState.cpp

```diff
@@ -397,6 +433,62 @@ void LSLuaState::finalizeAssemblyLoad(Assembly *assembly, utArray<Type *>& types
         }
     }
 
+    bool shrink = false;
```

Comment:
Can we comment what this and next loop does? It's not obvious by inspection but it's probably very important.